ushahidi/platform-client
# Feature/csv configure

This pull request makes the following changes:
- Added configure screen for bulk data import
- Split Post Editor directive into sub directives covering post status and post stages
- Added PostEditService to handle common functionality for post editing
- Added additional field fixed values for bulk data import, including all Form fields, tags, title and content
- Added Bulk Data Import mode for Post Editor
- Added switch to handle properties that should/shouldn't be enabled in Post Editor during Bulk Data Import mode

Test these changes by:
- Attempt to import a new csv
- Map the fields
- Set default values for all imported posts via the configure screen
- Check that the default values are correctly set for the newly imported Posts

Fixes ushahidi/platform# 945

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/173)
